docs/microservices.json

```diff
@@ -114,6 +114,10 @@
     {
       "name": "DARTS",
       "colour": "#3498DB"
+    },
+    {
+      "name": "AM",
+      "colour": "#3498DB"
```

Comment:
We need to pick a different colour to the others

Reply:
different colour chosen #ffcc66